From 462d48e979df3e9060afff2590f07fe919b399ce Mon Sep 17 00:00:00 2001
From: Alice Ryhl <aliceryhl@google.com>
Date: Wed, 10 Jun 2026 10:35:04 +0000
Subject: [PATCH 220/223] ANDROID: gki: use 1.82.0.p2 rustc

This respin of the Rust compiler fixes an issue in the uwtable
implementation that causes boot failures with KASAN on arm.

Bug: 517148969
Bug: 513032835
Change-Id: I99c948ce1c899668164a44a13937f6877e567ff7
(cherry picked from commit 3b4ccdd6098be7737ca151688915e9bd8fac3660)
Signed-off-by: Alice Ryhl <aliceryhl@google.com>
Bug: 522618399
---
 build.config.constants | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.config.constants b/build.config.constants
index 8143651801c7..ee563d819174 100644
--- a/build.config.constants
+++ b/build.config.constants
@@ -1,7 +1,7 @@
 BRANCH=android16-6.12
 KMI_GENERATION=6
 CLANG_VERSION=r536225
-RUSTC_VERSION=1.82.0.p1
+RUSTC_VERSION=1.82.0.p2
 AARCH64_NDK_TRIPLE=aarch64-linux-android31
 X86_64_NDK_TRIPLE=x86_64-linux-android31
 ARM_NDK_TRIPLE=arm-linux-androideabi31
-- 
2.50.1

